Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [GnuCash/Program](https://hosted.weblate.org/projects/gnucash/gnucash/).


It also includes following components:

* [GnuCash/Glossary](https://hosted.weblate.org/projects/gnucash/glossary/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/gnucash/gnucash/horizontal-auto.svg)

fellen commented on po/de.po:
```diff
@@ -5363,7 +5363,7 @@ msgstr "Konf. speichern"
 #: gnucash/gnome/gnc-plugin-page-report.cpp:242
 msgid "Save Config As…"
-msgstr "Konfiguration speichern unter…"
+msgstr "Konf. speichern unter…"
```
Warum abkürzen? Falls nötig, bitte Übersetzer-Kommentar verbessern, etwa <q>used on toolbar button</q>.

fellen commented on po/de.po:
```diff
@@ -30408,7 +30408,7 @@ msgid ""
 msgstr ""
 "Erstelle eine geplante Buchung mit der aktuellem Buchung als Vorlage oder "
-"oder bearbeite die geplante Buchung  aus der sie erzeugt wurde."
+"bearbeite die geplante Buchung aus der sie erzeugt wurde."
```
Buchung, aus

Ich bin mir auch nicht sicher, ob wir nicht besser bei terminiert bleiben.

fellen reviewed (commented):
@CWehli

mocsa committed:
Translation update  by mocsa <csaba@feltoltve.hu> using Weblate

po/hu.po: 75.2% (4192 of 5571 strings; 436 fuzzy)
5 failing checks (0.1%)
Translation: GnuCash/Program (Hungarian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/hu/

Co-authored-by: mocsa <csaba@feltoltve.hu>

SilentWarrior83 committed:
Translation update  by Arve Eriksson <031299870@telia.com> using Weblate

po/sv.po: 100.0% (5571 of 5571 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Program (Swedish)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/sv/

Co-authored-by: Arve Eriksson <031299870@telia.com>

CWehli committed:
Translation update  by Christian Wehling <christian.wehling@web.de> using Weblate

po/de.po: 100.0% (5571 of 5571 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Program (German)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/de/

Translation update  by Christian Wehling <christian.wehling@web.de> using Weblate

po/de.po: 100.0% (5571 of 5571 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Program (German)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/de/

Co-authored-by: Christian Wehling <christian.wehling@web.de>

fellen committed:
Translation update  by "Frank H. Ellenberger" <frank.h.ellenberger@gmail.com> using Weblate

po/de.po: 100.0% (5571 of 5571 strings; 0 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Program (German)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/de/

Co-authored-by: Frank H. Ellenberger <frank.h.ellenberger@gmail.com>

SziaTomi committed:
Translation update  by Szia Tomi <sziatomi01@gmail.com> using Weblate

po/hu.po: 75.3% (4197 of 5571 strings; 431 fuzzy)
3 failing checks (0.1%)
Translation: GnuCash/Program (Hungarian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/hu/

Translation update  by Szia Tomi <sziatomi01@gmail.com> using Weblate

po/hu.po: 75.3% (4196 of 5571 strings; 432 fuzzy)
3 failing checks (0.1%)
Translation: GnuCash/Program (Hungarian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/hu/

Translation update  by Szia Tomi <sziatomi01@gmail.com> using Weblate

po/hu.po: 75.2% (4194 of 5571 strings; 434 fuzzy)
3 failing checks (0.1%)
Translation: GnuCash/Program (Hungarian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/hu/

Translation update  by Szia Tomi <sziatomi01@gmail.com> using Weblate

po/hu.po: 75.2% (4194 of 5571 strings; 434 fuzzy)
3 failing checks (0.1%)
Translation: GnuCash/Program (Hungarian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/hu/

Translation update  by Szia Tomi <sziatomi01@gmail.com> using Weblate

po/hu.po: 75.2% (4194 of 5571 strings; 434 fuzzy)
3 failing checks (0.1%)
Translation: GnuCash/Program (Hungarian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/hu/

Translation update  by Szia Tomi <sziatomi01@gmail.com> using Weblate

po/hu.po: 75.2% (4194 of 5571 strings; 434 fuzzy)
3 failing checks (0.1%)
Translation: GnuCash/Program (Hungarian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/hu/

Translation update  by Szia Tomi <sziatomi01@gmail.com> using Weblate

po/hu.po: 75.2% (4193 of 5571 strings; 435 fuzzy)
4 failing checks (0.1%)
Translation: GnuCash/Program (Hungarian)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/hu/

Co-authored-by: Szia Tomi <sziatomi01@gmail.com>

soramikan committed:
Translation update  by soramikan <sora_yana@icloud.com> using Weblate

po/glossary/ja.po: 6.6% (14 of 212 strings; 1 fuzzy)
0 failing checks (0.0%)
Translation: GnuCash/Glossary (Japanese)
Translate-URL: https://hosted.weblate.org/projects/gnucash/glossary/ja/

Translation update  by soramikan <sora_yana@icloud.com> using Weblate

po/ja.po: 88.1% (4912 of 5571 strings; 311 fuzzy)
430 failing checks (7.7%)
Translation: GnuCash/Program (Japanese)
Translate-URL: https://hosted.weblate.org/projects/gnucash/gnucash/ja/

Co-authored-by: soramikan <sora_yana@icloud.com>

fellen commented:
Thanks!
